src/library/F3DStarter.h

```diff
@@ -15,7 +20,7 @@ class F3DStarter
   /**
    * Parse the options and configure a f3d::loader accordingly
    */
-  int Start(int argc, char** argv);
+  F3D_EXPORT int Start(int argc, char** argv);
```

Comment:
why does the methods needs to be exported for this one ?

Reply:
One solution to keep the unique_ptr in public API is to export each method instead of exporting the whole class.  
Exporting each method is too verbose in my opinion, but for this particular class it's temporary, so I didn't wanted to change the unique_ptr to a raw ptr and revert this change when the class is not in the public API anymore.

Reply:
got it.